Scoop/Supporting/Assets.xcassets/rideHistoryIcon.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "rideHistoryIcon.png",
+      "idiom" : "universal",
+      "scale" : "1x"
```

Comment:
Same as above. In Figma, you can export icons as 1x, 2x, and 3x. These should be labeled with the @ simple. For example, you will have a `rideHistoryIcon@1x.png`, `rideHistoryIcon@2x.png`, and `rideHistoryIcon@3x.png`. No need to worry about it for now, but something to keep in mind.